<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentitySettings</class>
 <widget class="QWidget" name="IdentitySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="publicGroup">
     <property name="title">
      <string>Public Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userName"/>
      </item>
      <item>
       <widget class="QLabel" name="statusMessageLabel">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="statusMessage"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="toxGroup">
     <property name="title">
      <string>Tox ID</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="CroppingLabel" name="toxIdLabel">
        <property name="text">
         <string>Your Tox ID (click to copy)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CroppingLabel</class>
   <extends>QLabel</extends>
   <header>src/widget/croppinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
